cmd/opentelemetry/cmd/all-in-one/main.go

```diff
@@ -115,6 +117,11 @@ func main() {
 	go func() {
 		err = svc.Run()
```

Comment:
```suggestion
		err := svc.Run()
```

Reply:
In this case why not 
```go 
handleErr(svc.Run())
```

Reply:
feel free to wrap them directly in handleErr() - always leave the code in a better state than what you found